content/docs/faq-state.md

```diff
@@ -78,6 +78,39 @@ handleSomething() {
 
 [Learn more about setState](/docs/react-component.html#setstate)
 
+### How do I update nested state properties?
+
+Let's say the state of your component looks like this:
+
+```jsx
+state = {
+    user: {
+        firstname: 'Bob',
+        lastname: 'Dylan',
+        age: 31
+    }
+}
+```
+
+If you want to update `user.lastname` while keeping the other properties the same, use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) in the update function to copy the original keys to a new object:
```

Comment:
There's no _spread operator_, it's the _spread **syntax**_ which you're correctly linking to already.